server/debug.lua

```diff
@@ -22,6 +22,7 @@ lib.addCommand('parsevehicles', {
 
             if topVehicleStats then
                 for vtype, data in pairs(topVehicleStats) do
+                    if not topStats then topStats = {} end
```

Comment:
https://github.com/overextended/ox_core/blob/main/client/debug.lua#L9

topStats is never nil (though may be an empty table, which doesn't matter).